generic/ltp.py.data/ltp-fs.yaml

```diff
@@ -0,0 +1,12 @@
+mem_leak: 0
```

Comment:
can we make this as default value? since it is zero in even other yaml files.
I even feel we can remove this parameter and make one variable in the test itself as sel.mem_leak = '0' and use the same variable in the test.

Reply:
default value is set to zero in ltp.py. I think its good to have mem_leak in yaml as well in case we want to change the value and run.
`self.mem_leak = self.params.get('mem_leak', default=0)`

Reply:
> default value is set to zero in ltp.py. I think its good to have mem_leak in yaml as well in case we want to change the value and run. `self.mem_leak = self.params.get('mem_leak', default=0)`

do we have any scenarios where we change this and test?

Reply:
currently no related to file system

Reply:
do we expect this in future?
if not then moving to a variable in test is the best way to reduce the yaml parameter so that it will be easy for even cfg runs.

Reply:
My take is we can keep this as   YAML variable is fine as this is LTP option which helps us to check mem-leak not exactly recalling what the use case as it was added via @harish-24

Reply:
@disgoel may you can remove this from your fs yaml

Reply:
addressed requested changes

Reply:
Thanks @disgoel  for incorporating the changes requested.